ts/features/walletV3/transaction/components/WalletTransactionInfoSection.tsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable functional/immutable-data */
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  Divider,
+  IOColors,
+  IORadiusScale,
+  IOVisualCostants,
+  ListItemHeader,
+  ListItemInfo,
+  ListItemInfoCopy,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import I18n from "../../../../i18n";
+import { Psp, Transaction } from "../../../../types/pagopa";
+import { format } from "../../../../utils/dates";
+import { clipboardSetStringWithFeedback } from "../../../../utils/clipboard";
+import TransactionReceiptDivider from "../../../../../img/features/wallet/transaction-receipt-divider.svg";
+
+type WalletTransactionInfoSectionProps = {
+  transaction?: Transaction | null;
+  psp?: Psp;
+  loading?: boolean;
+};
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: IOColors["grey-50"]
+  },
+  contentCard: {
+    borderRadius: IORadiusScale["1"],
+    marginVertical: IOVisualCostants.appMarginDefault
+  }
+});
+
+/**
+ * Component that shows a success message after the wallet onboarding process is completed
+ * TODO: Define the desired design of this component
+ */
+const WalletTransactionInfoSection = ({
+  transaction,
+  psp,
+  loading
+}: WalletTransactionInfoSectionProps) => (
+  <>
+    <TransactionReceiptDivider
+      height={24}
+      width={"100%"}
+      preserveAspectRatio="xMin slice" // Add this property to fit the width to the parent
+    />
+    <View
+      style={[
+        IOStyles.horizontalContentPadding,
+        styles.container,
+        IOStyles.flex
```

Comment:
I don't think it's the same, otherwise there should be another eslint rule that doesn't allow that. Anyway I provided to move this style into a single StyleSheet property just for more readability.

Addressed into 52ce78ba0217d146fdf7578127150c63dffaea19